File: GCFinalProject/ClientApp/src/app/CartoonChar.ts


export class CartoonChar {
    pkId: number;
    name: string;
    heavyAttack: number;
    lightAttack: number;

    constructor(pkId: number, name: string, heavyAttack: number, lightAttack: number){
      this.pkId = pkId;
      this.name = name;
      this.heavyAttack = heavyAttack;
      this.lightAttack = lightAttack;
    }
}

File: GCFinalProject/Models/CartoonChar.cs
using System;
using System.Collections.Generic;

namespace GCFinalProject.Models
{
    public partial class CartoonChar
    {
        public int PkId { get; set; }
        public string? Name { get; set; }
        public int? HeavyAttack { get; set; }
        public int? LightAttack { get; set; }
    }
}
